README.md

```
@@ -308,6 +308,160 @@ const toolResult = await tool.invoke(
+			}
+		} as MaximVercelProviderMetadata
+	}
+});
+```
+
+#### Complete example
+
+```ts
+const logger = await maxim.logger({ id: repoId });
+if (!logger) {
+	throw new Error("Logger is not available");
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)